uni-snd/src/web_snd.rs

```diff
@@ -0,0 +1,89 @@
+use stdweb;
+use stdweb::unstable::TryInto;
+use stdweb::web::TypedArray;
+
+use uni_app::App;
+
+use super::SoundGenerator;
+
+pub struct SoundDriver<T> {
+    generator: Option<Box<SoundGenerator<T>>>,
+    ctx: stdweb::Value,
+    start_audio: f64,
+    buffer: [f32; BUFFER_SIZE as usize * 2],
+}
+
+const BUFFER_SIZE: u32 = 2048;
+const AUDIO_LATENCY: f64 = 0.1;
+
+impl<T> SoundDriver<T> {
```

Comment:
Should it be also bound by `T: Send + 'static` too ? Although the SoundGenerator itself is bound by Send, but it is better to make webgl one interface same as native one ? But Im not so sure about that...

Reply:
There's no need for Send on the web side and I don't know what limitations this implies on the sound generator, so well I'm not sure neither. I'll try and see if it still compiles with Send on wasm32 target

Reply:
Okay then, at least the SoundGenerator is bound by Send, which prevent users who always use wasm32-target for development, suddenly find that code is not working (using a non-Send type) when he switch to normal cargo run target.